experimental/dds/tree2/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -990,68 +984,78 @@ describe("SharedTree", () => {
 	describe("Events", () => {
 		it("triggers events for local and subtree changes", () => {
 			const view = testTreeView();
-			const root = view.context.root.getNode(0);
+			const rootNode = view.context.root.getNode(0);
+			const root = view.root as unknown as { x: number };
```

Comment:
@CraigMacomber I did this casting to avoid having to refactor the schema stuff out of testTreeView. Is this reasonable?

Reply:
its fine I guess.